frontend/src/components/LoginAndRegistrationPage/LoginAndRegistrationPage.tsx

```diff
@@ -31,7 +32,7 @@ const LoginAndRegistrationPage: React.FC<{}> = () => {
                                     <LoginPage/>
                                 </Tab.Pane>
                                 <Tab.Pane eventKey="registration">
```

Comment:
Fajnie by było mieć opcję nawigowania do rejestracji lub logowania a nie do strony login-and-registration (nawet w linku to źle wygląda), nie wiem jak to można podzielić ale może coś wymyślisz.